db/seeds.rb

```diff
@@ -5,3 +5,38 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+# User.destroy_all
+# Aircraft.destroy_all
+# puts "Cleaning Database"
+# puts "Creating Users and their aircrafts"
+
+# 5.times do
+#   user = User.create!(
+#     first_name: Faker::Name.first_name,
+#     last_name: Faker::Name.last_name,
+#     email: Faker::Internet.email,
+#     password: Faker::Internet.password
+#   )
+# end
+
+## Need to manually increment user id to assign an aircraft to a user
+# aircraft = Aircraft.create!(
+#   title: Faker::Lorem.sentence,
+#   description: Faker::Lorem.paragraph(sentence_count: 2, supplemental: false, random_sentences_to_add: 4),
+#   aircraft_type: Faker::Space.launch_vehicle,
+#   price_per_day: rand(150..2000),
+#   max_occupants: rand(4..30),
+#   departure_address: Faker::Address.city,
+#   arrival_address: Faker::Address.city,
+#   user_id: 11
+# )
+# puts "finished!"
+
+booking = Booking.create!(
+  start_date: DateTime.strptime("23/12/2022 11:00", "%d/%m/%Y %H:%M"),
+  end_date: DateTime.strptime("26/12/2022 11:00", "%d/%m/%Y %H:%M"),
+  total_price: (729*3),
+  user_id: 8,
```

Comment:
don't hardcode user_id data, assign a user to variable then take that instance variable;s id. Same with aircraft id !

Reply:
If you can show me how to write the code that allows this to happen randomly when generating users on a new seed, would appreciate it!